spark/interpreter/pom.xml

```diff
@@ -63,6 +61,12 @@
       <version>${project.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.zeppelin</groupId>
+      <artifactId>spark-scala-2.10</artifactId>
+      <version>${project.version}</version>
```

Comment:
doesn't this break if/when spark 2.4.0 doesn't have a spark-scala-2.10?

Reply:
spark-scala-2.10 will always compile with spark 2.2 and scala 2.10. The profile only affect the embedded spark version

Reply:
spark-scala-2.10 will always compile with spark 2.2 and scala 2.10. The profile only affect the embedded spark version

Reply:
well, ok, but that should go to spark 2.4 at one point? or it should be under a profile so people can choose the version of spark they want in embedded?

Reply:
People can still choose profile spark-2.4 so that the embedded spark version will be 2.4